client/src/pages/LogInForm.js

```diff
@@ -25,19 +25,21 @@ function LogInForm() {
 			const data = await post("/api/sign-in", userData);
 			const { token, user_type } = data.resource;
 
-				localStorage.setItem("authToken", token);
-				localStorage.setItem("userType", user_type);
+			localStorage.setItem("authToken", token);
+			localStorage.setItem("userType", user_type);
 
-				switch (user_type) {
-					case "admin":
-						navigate("/admin-dashboard");
-						break;
-					case "buyer":
-						navigate("/buyer-dashboard");
-						break;
-					default:
-						navigate("/bidder-dashboard");
-				}
+			switch (user_type) {
+				case "admin":
+					navigate("/admin-dashboard");
+					break;
+				case "buyer":
+					navigate("/buyer-dashboard");
+					break;
+				default:
+					navigate("/bidder-dashboard");
+			}
+
+			navigate("/list-tenders");
```

Comment:
This is not related to this commit. Create another commit that fixes lint checks.

And please make sure your pre-commit check is working. This should never happen